internal/host/host.go

```diff
@@ -1174,7 +1174,7 @@ func (m *Manager) captureConnectivityReportMetrics(ctx context.Context, openshif
 	log := logutil.FromContext(ctx, logrus.New())
 	defer commonutils.MeasureOperation("CaptureConnectivityReportMetrics", log, m.metricApi)
 
-	connectivityReport, err := hostutil.UnmarshalConnectivityReport(h.Connectivity)
+	connectivityReport, err := hostutil.UnmarshalConnectivityReport(report)
```

Comment:
Is there a test that can be added to verify it?

Reply:
I'll add one.

Reply:
Done. Can you review again, please?